tools/alpine/versions.aarch64

```diff
@@ -1,370 +1,387 @@
-# linuxkit/alpine:6f9efe87b3c394196949985f0ed6b5263432670a-arm64
+# linuxkit/alpine:1e9d030f156213bf857e32cb824e86a834e29d5f-
```

Comment:
looks like the arch is missing here. I thought this is handled automatically by the build/makefile

Reply:
So did I. How odd. The really frustrating thing is that this is in the very first step. I loathe the idea of running through the whole thing again, having done it 3 times. First to update the first `FROM`, then when I realized we `FROM` again later, and then because of a bug in `go-compile.sh`. I will look into it, though.

This does not pass CI yet.

Reply:
It did the right thing on `x86_64`. Huh

Reply:
Ah shoot. The arm64 builder was not running that build in buildx. All other packages we build with `linuxkit pkg build`, which ensures bulldkit in a container, but `tools/alpine` is built directly with `docker build`, which doesn't have build if older. I need to upgrade docker there.

Reply:
Done. Now it includes it.